addons/mail/static/src/models/thread/thread.js

```diff
@@ -561,11 +561,16 @@ function factory(dependencies) {
             ) {
                 return;
             }
+            const message = this.env.models['mail.message'].findFromIdentifyingData({ id: messageId });
+            if (message.isTransient) {
+                this.update({ lastSeenByCurrentPartnerMessageId: messageId });
+                return;
+            }
```

Comment:
Shouldn't we actually just convert id to `Math.floor(messageId)` at the start of the method, and let the rest of the code work normally?

- I don't think it's a good idea to leak the float id in `lastSeenByCurrentPartnerMessageId`.
- And we might need to actually call message seen with the last real id, even in this case?

Reply:
That's what was done and the task is saying 

> It's impossible to mark command message as read due to its fake ID, and the RPC to message_seen had to be fixed with Math.floor but it shouldn't be done instead.

So what is the purpose of this PR then ?

Reply:
In regard to this method, moving `floor` at the start instead of at the end?

Other fixes are still necessary too (auto scroll, transient message not considered when computing unread, ...)